add-on/_locales/en-US/messages.json

```diff
@@ -382,5 +382,13 @@
   "mixed_content_desc": {
     "message": "There should be no mixed content on MDN",
     "description": "Description of Mixed Content rule"
+  },
+  "anchor_exists": {
+    "message": "Anchor exists",
```

Comment:
I think the labels are always describing problems. So this would be "Anchor doesn't exist"